rclcpp/include/rclcpp/node.hpp

```diff
@@ -359,11 +360,24 @@ class Node : public std::enable_shared_from_this<Node>
     rclcpp::event::Event::SharedPtr event,
     std::chrono::nanoseconds timeout);
 
+  RCLCPP_PUBLIC
+  rclcpp::Clock::SharedPtr
+  get_clock(rcl_clock_type_t clock_type = RCL_ROS_TIME);
+
+  RCLCPP_PUBLIC
+  Time
+  now(rcl_clock_type_t clock_type = RCL_ROS_TIME);
```

Comment:
Why does this need the clock type? Shouldn't it just always be ROS time?

Rather than having a parameter (which is an exposed rcl type rather than a native rclcpp construct) to choose the clock type, it would be great (imo) if there were just different functions, like `now` and `wall_now` or something. Honestly, the only thing the Node needs to be able to return is ROS time in my opinion. If the use wants system time only, then they can create a clock call now on it or use std::chrono, there's no need for a node. I feel the same way about the `get_clock` function, ideally it would always the ROS time clock. I understand the convenience of getting any of the clocks back, but honestly I think having a separate type for steady clock is still desirable.

Reply:
Yeah, I definitely agree with the steady clock being a different type. With that we'll need a different function, and so we might as well switch to three different methods. I'll go ahead and switch to that instead of the parameter.

Reply:
Or I could just take your suggestion of only providing the ROS clock and let power users use the Clock API directly if they want system or steady time. I think that might make sense as a conservative first step. We can always add support for the others later.

Reply:
Sounds good to me.